types/react-paginate/index.d.ts

```diff
@@ -1,11 +1,8 @@
-// Type definitions for react-paginate 4.3
+// Type definitions for react-paginate 5.2
 // Project: https://github.com/AdeleD/react-paginate
-// Definitions by: Simon Hartcher <https://github.com/deevus>
-//                 Wouter Hardeman <https://github.com/wouterhardeman>
-//                 pegel03 <https://github.com/pegel03>
-//                 Simon Archer <https://github.com/archy-bold>
+// Definitions by: Yasunori Ohoka <https://github.com/yasupeke>
```

Comment:
Can you just add yourself to the list of owners instead of replacing the existing list?